django-backend/fecfiler/contacts/views.py

```diff
@@ -39,6 +38,28 @@ def validate_and_sanitize_candidate(candidate_id):
     return candidate_id
 
 
+def save_payload_contacts(payload, contact_keys, request):
```

Comment:
I'm seeing overlap between `save_payload_contacts` and `create_or_update_contact`.  Seems like `write_contacts` =  `create_or_update_contact` X `contact_keys` = `save_payload_contacts` but in the serializer instead of the view.  I wonder if we can share the code